### PR TITLE
Change animation speed scale of a sprite object

### PR DESCRIPTION
**Pull request by victorlevasseur**
Add an action to change the animation speed scale of a sprite object and an expression to retrieve the speed scale.
There is also an example in which the player object accelerates when the player continues pressing the movement keys.

### CODE REVIEW

File: GDCpp/GDCpp/RuntimeSpriteObject.h [hunk 1]
```diff
@@ -164,7 +167,9 @@ public :
     float currentAngle;
     unsigned int currentSprite;
     bool animationStopped;
+
     float timeElapsedOnCurrentSprite;
+    float AnimationSpeedScale;
```
Comment (4ian) [inline GDCpp/GDCpp/RuntimeSpriteObject.h]: Could you just change the member name to camelCase instead of CamelCase? ;)

**Comment by victorlevasseur** [comment]
Fixed.

**Comment by 4ian** [comment]
Excellent as usual! :+1:

**Comment by BillyOutlast** [comment]
@dependabot rebase

**Comment by BillyOutlast** [comment]
@dependabot recreate

**Pull request merged**